inspec/a2-iam-v2p1-only-integration/controls/introspection.rb

```diff
@@ -1,81 +0,0 @@
-# encoding: utf-8
```

Comment:
This was a copy-pasted file. Now that `a2-iam-v2p1-only-integration` has been combined it is no longer needed.